client-src/default/utils/log.js

```diff
@@ -26,17 +21,13 @@ export function setLogLevel(level) {
     case TRACE:
       log.setLevel(level);
       break;
-    // deprecated
-    case WARNING:
-      // loglevel's warning name is different from webpack's
-      log.setLevel('warn');
-      break;
-    // deprecated
-    case NONE:
     case SILENT:
       log.disableAll();
       break;
     default:
+      if (level === 'none' || level === 'warning') {
+        log.error('[WDS] "none" and "warning" have been deprecated.');
+      }
```

Comment:
Remove this, no need this in next release, it is major release

Reply:
fm, I want to avoid receiving many issues about these changes from everyone.😞

Reply:
ok, I'll delete here but I'll add warn to master. what do you think?